dev-python/pyside6/pyside6-6.4.2.ebuild

```diff
@@ -87,7 +87,7 @@ RDEPEND="${PYTHON_DEPS}
 		x11-libs/libxkbcommon
 	)
 	help? ( =dev-qt/qttools-${QT_PV}[assistant] )
-	multimedia? ( =dev-qt/qtmultimedia-${QT_PV}[qml(+)?,gles2-only(-)=,widgets(+)?] )
```

Comment:
I'd `git mv` for this to avoid confusion as I expect a fair number of people have unmasked Qt 6.